verilog/formatting/align.cc

```diff
@@ -720,8 +720,41 @@ class DataDeclarationColumnSchemaScanner : public VerilogColumnSchemaScanner {
         }
         break;
       }
-      case NodeEnum::kDimensionRange:
-      case NodeEnum::kDimensionScalar:
+      case NodeEnum::kDeclarationDimensions: {
+        if (current_path_ == SyntaxTreePath{1, 0, 3, 0}) {
+          SyntaxTreePath new_path{1, 0, 3};
+          const ValueSaver<SyntaxTreePath> path_saver(&current_path_, new_path);
+          TreeContextPathVisitor::Visit(node);
+          return;
+        }
+        break;
+      }
+      case NodeEnum::kDimensionScalar: {
+        CHECK_EQ(node.children().size(), 3);
```

Comment:
Are there ways to react more gracefully if this condition is not true ?
Fine for now, but it might be useful to fine-comb through all our `CHECK`s and see if they never can be triggered by user data bu only by programming errors,
(as in the language server, we have verible running in a long-running process that will see a lot of invalid code pass by - and we want to avoid crashing there.